### DISCUSSION TITLE
[Bug] How to pass the touch from children to parent

### DISCUSSION THREAD

**Discussion by Sasikumar3595** [dotnet/maui#3274] [Q&A]
### Description

I was create the custom layout that have pan gesture and i was add the scroll view as it child. how to pass the touch to custom layout when scroll view reaches it end position.

### Expected Behavior

- Custom layout(parent) pan gesture will get the touch when the scroll view reaches to its end.

### Actual Behavior

- Custom layout(parent) pan gesture does not get the touch.

**Comment by Sasikumar3595** [comment]
Is there possible to pass the touch to parent from its child(scroll view), when it scroll ends?

**Comment by Nijamudeen-Mohamed** [comment]
We require the same scenario, we would like to pass touch to scroll view parent. For example, I have the Following UI structure,

**AbsoluteLayout** [sliding horizontal] ->**ScrollView** [Performs vertical scroll] -> **Grid** [Containing 3 DrawableViews]

The issue is, ScrollView does not pass its touch to AbsoluteLayout to perform the sliding operation.
